### ISSUE TITLE
Make the lerna lib build run in parallel to improve speed

### ISSUE THREAD

**Issue by SimeonC** [tablecheck/tablecheck-react-system#24]
Referring to this particular part of the build lib process: https://github.com/tablecheck/tablecheck-react-system/blob/bee6ff77f71c1b3bea9150191502f36240da3b1c/packages/scripts/scripts/build-lib.js#L38-L44

2 things that would need to be fixed;
* references needs to be a dependency graph, so we don't try build one package before it's dependencies are all built.
* The `logTaskEnd` and `logTaskStart` would need to handle this properly or the log output would end up getting out of order and messed up.

**Comment by SimeonC** [comment]
NX does this better so we won't do this anymore
